tests/integration/actions/config/test_direct_interactive_ee.py

```diff
@@ -0,0 +1,32 @@
+""" config direct from cli interactive w/ ee
+"""
+import pytest
+
+from .base import BaseClass
+
+from ..._common import get_executable_path
+from ..._common import container_runtime_or_fail
+
+CLI = (
+    f"{get_executable_path('python')}"
```

Comment:
Similar to below - I'd either merge this with the f-string below, or drop the f-string syntax from this line and just leave the function call to return a string.